tap-gui/tap-gui-rbac/setup-tap-gui-rbac-eks.md

```diff
@@ -0,0 +1,157 @@
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+
+
+2. Using `eksctl`, execute the following command:
+
+```console
+eksctl associate identityprovider -f rbac-setup.yaml
+```
+
+3. To check if the association of the OIDC provider with the EKS cluster was successful, run the following command
+
+```console
+eksctl get identityprovider --cluster `CLUSTER-NAME`
+```
+
+The output should state `ACTIVE` in the STATUS column.
+
+
+## <a id="configure-tap-gui"></a> Configure the Tanzu Application Platform GUI
+
+The next step is to configure visibility of the remote cluster in Tanzu Application Platform GUI. 
+
+1. Make sure you added an `auth` section to the `app_config` file taht Tanzu Application Platform GUI uses. In the example for Auth0, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+auth:
+    environment: development
+    providers:
+        autho:
+            development:
+                clientId: "CLIENT-ID"
+                clientSecret: "CLIENT-SECRET"
+                domain: "ISSUER-URL"
+```
+Where:
+
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-SECRET` is the Client Secret you obtained while setting up the OIDC provider
+
+
+2. You must also add a `kubernetes` section to the `app_config` file that Tanzu Application Platform GUI uses. This section must have an entry for each cluster that has resources to view.
+
+To do so, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+kubernetes:
+    serviceLocatorMethod:
+        type: 'multiTenant'
+    clusterLocatorMethods:
+      - type: 'config'
+        clusters:
+          - name: "CLUSTER-NAME"
+            url: "CLUSTER_URL"
+            authProvider: oidc
+            oidcTokenProvider: auth0
+            skipTLSVerify: true
+            skipMetricsLookup: true
+```
+Where:
+
+   - `CLUSTER-NAME` is the cluster name for your EKS cluster
+   - `CLUSTER_URL` is the URL for the remote cluster you are connecting to Tanzu Application Platform GUI. To obtain your cluster's URL, execute the following command:
+
+```console
+CLUSTER_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+
+echo CLUSTER_URL: $CLUSTER_URL
+```
+
+If there are any other clusters that you would like to make visible through Tanzu Application Platform GUI, add their entries to `clusters` as well.
+
+## <a id="upgrade-tap-gui"></a> Upgrade the Tanzu Application Platform GUI package
+
+Once the new configuration file is ready, update the `tap-gui` package by running this command:
+
+```console
+tanzu package installed update tap-gui --values-file tap-gui-values.yaml
+```    
+
+Or, if you were updating the `tap-values.yml` file, run the following command: 
+
+```console
+tanzu package installed update tap --values-file tap-values.yml
+```
```

Comment:
I'd like to see one of two things happen here:
1. We only talk about profile-based installs, with no mention of `tap-gui-values.yaml`
2. We use some standard terminology like "installing via a TAP profile" and "installing TAP GUI as a standalone/individual package" - because I think distinguishing these by the filename of the YAML file will be confusing for an uninformed reader.

Reply:
Thanks,  @andreizimin addressed this in a separate PR.